client/mass/charts.js

```diff
@@ -286,9 +323,109 @@ function setRenderers(self) {
 		})
 	}
 
+	self.showGenesetEditUI = async chart => {
```

Comment:
can comment that right now, this ui is hardcoded to produce geneExpression terms. later may generalize it for other gene-based quantification data